src/sagemaker/jumpstart/types.py

```diff
@@ -1001,6 +1007,206 @@ def __init__(
         self.id_info = id_info
 
 
+class HubContentDependency(JumpStartDataHolderType):
```

Comment:
I actually didn't know we had this in the response :) Curious to see how we can leverage this in the copy workflow